src/Render/NeoFormRenderer.php

```diff
@@ -132,7 +132,7 @@ public function formRow($el, array $attrs = []): Html
     public function formInput(BaseControl $el, array $attrs = []): Html
     {
         if ($attrs['readonly'] ?? (bool)$el->getOption('readonly')) {
-            return $this->template($el->getForm())->readonly($el);
+            $el->setAttribute('readonly', true);
```

Comment:
If we need to have this functionality, maybe we could have a different option name for it? This seems kind of a breaking change.